src/cryptonote_core/service_node_list.cpp

```diff
@@ -1145,20 +1146,23 @@ namespace service_nodes
     // Check node contributor counts
     {
       bool too_many_contributions = false;
-      if (hf_version >= cryptonote::network_version_16_pulse)
+      if (hf_version >= cryptonote::network_version_19)
+        // As of HF19 we allow up to 10 stakes total
+        too_many_contributions = existing_contributions + stake.locked_contributions.size() > MAX_NUMBER_OF_CONTRIBUTORS_V2;
```

Comment:
Should this not be:
```suggestion
        too_many_contributions = existing_contributions + other_reservations + 1 > MAX_NUMBER_OF_CONTRIBUTORS_V2;
```
to match the logic under HF16 (but just changing the RHS variable)?